### PR TITLE
Nrfx 4806 move timer strobe fix from shell to npmx

### PR DESCRIPTION
**Pull request by wilu-nordic**
test:PR-201

### CODE REVIEW

File: samples/timer_wakeup/src/main.c [hunk 1]
```diff
@@ -162,9 +162,6 @@ void main(void)
 	npmx_led_mode_set(npmx_led_get(npmx_inst, LED_IDX), NPMX_LED_MODE_HOST);
 	npmx_led_state_set(npmx_led_get(npmx_inst, LED_IDX), false);
 
-	/* Save the compare value data to internal timer's registers. */
-	npmx_timer_task_trigger(m_timer_inst, NPMX_TIMER_TASK_STROBE);
-
```
Comment (magp-nordic) [inline samples/timer_wakeup/src/main.c:167]: Shouldn't call to `npmx_timer_config_set` be also moved below `npmx_led_state_set`. Now, led config is set after the timer config is applied, not before, like it used to be.

Comment (wilu-nordic) [reply samples/timer_wakeup/src/main.c:167]: I checked and the sample is working as before.

**Review by magp-nordic — APPROVED** [review]

**Comment by nordic-segl** [comment]
Integration CI is failing on
uart:~$ npmx ldsw soft_start current set 1 75
Error: unable to convert milliamperes to soft-start current.
uart:~$ npmx ldsw soft_start current set 0 25
Error: unable to convert milliamperes to soft-start current.

modules/npmx needs rebase???

**Comment by wilu-nordic** [comment]
> Integration CI is failing on uart:~$ npmx ldsw soft_start current set 1 75 Error: unable to convert milliamperes to soft-start current. uart:~$ npmx ldsw soft_start current set 0 25 Error: unable to convert milliamperes to soft-start current.
> 
> modules/npmx needs rebase???

Rebased.

**Review by magp-nordic — APPROVED** [review]

**Review by masz-nordic — APPROVED** [review]

**Review by nordic-segl — APPROVED** [review]

**Pull request merged**